src/main/resources/google_checks.xml

```diff
@@ -45,7 +45,8 @@
   <module name="LineLength">
     <property name="fileExtensions" value="java"/>
     <property name="max" value="100"/>
-    <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
+    <property name="ignorePattern"
+             value="^package.*|^import.*|&lt;a href=&quot;(.*?)&quot;&gt;(.*?)&lt;\/a&gt;|href|http://|https://|ftp://"/>
```

Comment:
remove `&lt;a ` and `&lt;\/a&gt;`

Reply:
https://github.com/checkstyle/checkstyle/pull/14943#discussion_r1631183000

Reply:
reverted all these changes.